README.md

```diff
@@ -18,4 +19,4 @@ To contribute to :
 - The owner of the repository takes care of accepting the pull request.
 
 Useful reminder for gitflow: 
-https://danielkummer.github.io/git-flow-cheatsheet/
\ No newline at end of file
+https://danielkummer.github.io/git-flow-cheatsheet/
```

Comment:
There is no changement. Useless change of life sequence.

Reply:
Sorry, can you merge with this mistake ? Thanks